src/test/java/com/google/jenkins/plugins/computeengine/ConfigAsCodeTest.java

```diff
@@ -51,6 +51,7 @@ public void shouldCreateCloudInstanceFromCode() throws Exception {
     assertEquals("Wrong configurations numExecutors", "1", configuration.getNumExecutorsStr());
     assertEquals("Wrong configurations runAsUser", "jenkins", configuration.getRunAsUser());
     assertEquals("Wrong configurations remoteFs", "agent", configuration.getRemoteFs());
+    assertEquals("Wrong configurations javaExecPath", "java", configuration.getJavaExecPath());
```

Comment:
I think it would be good to also add an integration test which makes use of a non-standard Java exec path. It could mean being able to change the startup script which might not be trivial, but an easier thing to do might just be to use the full path of the java that already gets installed instead of the default "java" and making sure that works as well.

Reply:
Startup script could work since currently the linux IT's requires the startup script to install Java.

Reply:
Done. Thanks @rachely3n for the suggestion, that really helped :)